packages/amplication-client/src/ServiceConnections/topics/TopicsList.tsx

```diff
@@ -57,10 +57,12 @@ export default function TopicsList({
         )}
       />
     ) : (
-      <EmptyState
-        message="This message broker has no topics"
-        image={EnumImages.CommitEmptyState}
-      />
+      <div style={{ padding: '15.25%' }}>
+        <EmptyState
+          message="This message broker has no topics"
+          image={EnumImages.CommitEmptyState}
+        />
+      </div>
```

Comment:
 it is better to set the height of the parent to 100% (depends on its parent) instead of using specific numbers. 
Also, the style should be in the SCSS file instead of inline

Reply:
height 100% adds a scrollbar to the side. I can add the style to scss file.
 The padding is in percentage. If you think this approach is not good then I can try something else.